includes/REST_API/Stories_Controller.php

```diff
@@ -249,6 +249,7 @@ public function filter_posts_clauses( $clauses, $query ) {
 	 * Retrieves a collection of web stories.
 	 *
 	 * @SuppressWarnings(PHPMD.NPathComplexity)
+	 * @SuppressWarnings(PHPMD.CyclomaticComplexity)
 	 * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
```

Comment:
A strong indicator that this method should be refactored and perhaps split up into a few smaller ones.